capstone/capweb/views.py

```diff
@@ -58,16 +65,29 @@ def contact(request):
 
     return render(request, 'contact.html', {
         "form": form,
-        "email": settings.DEFAULT_FROM_EMAIL
+        "email": settings.DEFAULT_FROM_EMAIL,
+        'page_image': 'img/og_image/contact.png',
+        'page_title': 'Contact Caselaw Access Project',
+        'page_description': 'Email us at info@example.com or fill out this form. '
```

Comment:
I think we want `settings.DEFAULT_FROM_EMAIL` instead of `info@example.com` here ...